packages/lit-dev-content/src/components/litdev-search.ts

```diff
@@ -393,28 +391,31 @@ export class LitDevSearch extends LitElement {
   private _onInput(e: InputEvent) {
     this._searchText = (e.target as HTMLInputElement).value ?? '';
     this._querySearch(this._searchText);
+    this._selectedIndex = -1;
   }
 
   /**
    * Populate suggestion dropdown from query.
    *
    * An empty query clears suggestions.
    */
-  private _querySearch(query: string) {
+  private async _querySearch(query: string): Promise<Suggestion[]> {
```

Comment:
Can this be inlined into the Task definition?

Reply:
not applicable anymore